app/ldap.go

```diff
@@ -73,7 +73,7 @@ func (a *App) GetAllLdapGroupsPage(page int, perPage int, opts model.LdapGroupSe
 		}
 	} else {
 		ae := model.NewAppError("GetAllLdapGroupsPage", "ent.ldap.app_error", nil, "", http.StatusNotImplemented)
-		mlog.Error(fmt.Sprintf("%v", ae.Error()))
+		mlog.Error("Unable to use ldap", mlog.Any(opts))
```

Comment:
I'm not sure it makes sense to log `opts` here, at least not without `page` and `perPage`. I'd suggest just logging the original error:
```suggestion
		mlog.Error("Unable to use ldap", mlog.Err(ae))
```